web-common/src/features/dashboards/pivot/pivot-queries.ts

```diff
@@ -0,0 +1,162 @@
+import type {
+  PivotAxesData,
+  PivotDataStoreConfig,
+} from "@rilldata/web-common/features/dashboards/pivot/types";
+import type { StateManagers } from "@rilldata/web-common/features/dashboards/state-managers/state-managers";
+import { sanitiseExpression } from "@rilldata/web-common/features/dashboards/stores/filter-utils";
+import { useTimeControlStore } from "@rilldata/web-common/features/dashboards/time-controls/time-control-store";
+import type { TimeRangeString } from "@rilldata/web-common/lib/time/types";
+import {
+  V1Expression,
+  V1MetricsViewAggregationDimension,
+  V1MetricsViewAggregationResponseDataItem,
+  V1MetricsViewAggregationSort,
+  createQueryServiceMetricsViewAggregation,
+  type V1MetricsViewAggregationResponse,
+} from "@rilldata/web-common/runtime-client";
+import type { CreateQueryResult } from "@tanstack/svelte-query";
+import { Readable, derived, readable } from "svelte/store";
+
+/**
+ * Wrapper function for Aggregate Query API
+ */
+export function createPivotAggregationRowQuery(
+  ctx: StateManagers,
+  measures: string[],
+  dimensions: V1MetricsViewAggregationDimension[],
+  whereFilter: V1Expression,
+  sort: V1MetricsViewAggregationSort[] = [],
+  limit = "100",
+  offset = "0",
```

Comment:
Why are these strings?

Reply:
The type for these from orval is `string`. The swagger spec says it is string as well. Not sure if this is done intentionally.